src/main/resources/jenkins/plugins/office365connector/workflow/Office365ConnectorSendStep/help.html

```diff
@@ -1,9 +1,3 @@
-<div>
-  This is the main text in a message attachment, and can contain standard message markup.
-  The content will automatically collapse if it contains 700+ characters or 5+ linebreaks, and will display a "Show
-  more..." link to expand the content.
-  Notification may include global variables, for example environment and currentBuild variables:<br>
-  <code>
-    office365ConnectorSend message:"started ${env.JOB_NAME} ${env.BUILD_NUMBER} (<${env.BUILD_URL}|Open>)"
-  </code>
+<div align="help">
+  Step for sending build notifications to Microsoft Teams or Outlook.
```

Comment:
by this we lost some example which might be helpful for newbies

Reply:
all of the deleted text is also available in help file for "message" field - so it's not lost.
If you want, I could split this text into those 2 files (help.html and help-message.html)
so the help.html would include the general information, and the help-message.html would describe only the "message" part.